prusti-common/src/vir/ast/expr_transformers.rs

```diff
@@ -695,6 +740,35 @@ pub trait FallibleExprFolder: Sized {
     ) -> Result<Expr, Self::Error> {
         Ok(Expr::SnapApp(self.fallible_fold_boxed(e)?, p))
     }
+
+    fn fallible_fold_container_op(
+        &mut self,
+        kind: ContainerOpKind,
+        l: Box<Expr>,
+        r: Box<Expr>,
+        p: Position,
+    ) -> Result<Expr, Self::Error> {
+        Ok(Expr::ContainerOp(
+            kind,
+            self.fallible_fold_boxed(l)?,
+            self.fallible_fold_boxed(r)?,
+            p,
+        ))
+    }
+
+    fn fallible_fold_seq(
+        &mut self,
+        ty: Type,
+        elems: Vec<Expr>,
+        p: Position,
+    ) -> Result<Expr, Self::Error> {
+        let mut folded = Vec::with_capacity(elems.len());
+        for e in elems {
+            folded.push(self.fallible_fold(e)?);
+        }
+
+        Ok(Expr::Seq(ty, folded, p))
```

Comment:
```suggestion
        Ok(Expr::Seq(
            ty,
            elems.into_iter()
                .map(self.fallible_fold)
                .collect::<Result<_>>()?,
            p,
        ))
```

Reply:
oh, neat, didn't know collect had `Control.Monad.sequence` built-in :D